### PR TITLE
Tests and Increased coverage

### PR DESCRIPTION
**Pull request by fabiorphp**
* Roles
* Renderers
* Controllers
* Models

### CODE REVIEW

File: src/Phansible/Controller/BundleController.php [hunk 1]
```diff
@@ -124,4 +123,31 @@ protected function outputBundle($zipPath, Application $app, $filename)
 
         return $response;
     }
+
+    /**
+     * @return \Phansible\Model\VagrantBundle
+     */
+    private function getVagrantBundle()
```
Comment (naxhh) [inline src/Phansible/Controller/BundleController.php]: Instead of doing this try to inject the bundle in the constructor.

Comment (fabiorphp) [inline src/Phansible/Controller/BundleController.php]: Do you have any tips to make this work?

I don't know if the Flint Framework have any support for this feature.

Comment (naxhh) [inline src/Phansible/Controller/BundleController.php]: As flint is silex with cute things.. should be pretty easy.

But I don't really know. So let it as is and I'll check that for another pr

**Comment by naxhh** [comment]
I'm not a fan of getter's testing, and I'm not really in favor of 100% coverage just because....

Besides that. I think the PR is ok. And if you all think getters testing is needed, we will do it.
Waiting more reviewers.

**Pull request merged**